### PR TITLE
Updated links

### PR DESCRIPTION
**Pull request by Linda-Editor**
1. Replaced graph.microsoft.io with appropriate target
2. Updated all instances of OData Query Parameters to point to https://developer.microsoft.com/graph/docs/concepts/query_parameters (there were various renditions)
3. Removed en-us from developer.microsoft.com and msdn.microsoft.com links
4. Changed http:// to https://
5. Updated all Graph links on Change Log to relative links (../api..., ../concepts, etc.)
6. Updated other links on Change Log to actual targets

### CODE REVIEW

**Comment by Linda-Editor** [comment]
@Lauragra I created a fresh fork and then redid all the changes, so this should be good now...

File: api-reference/beta/resources/privacyprofile.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Represents a company's privacy profile, which includes a privacy statement URL a
 | Property   | Type|Description|
 |:---------------|:--------|:----------|
 |contactEmail|String| A valid smtp email address for the privacy statement contact. Not required.|
-|statementUrl|String| A valid URL format that begins with http:// or https://. Maximum length is 255 characters. The URL that directs to the company's privacy statement. Not required.|
+|statementUrl|String| A valid URL format that begins with https:// or https://. Maximum length is 255 characters. The URL that directs to the company's privacy statement. Not required.|
```
Comment (Lauragra) [inline api-reference/beta/resources/privacyprofile.md]: Please look for instances like this where the replace wasn't logical.

Comment (Linda-Editor) [reply api-reference/beta/resources/privacyprofile.md]: @Lauragra Thanks for catching that. I fixed instances of that in two files, and then tried to find similar ones, but could not find any others. Let me know if you see anything else.

**Comment by Lauragra** [comment]
Great! I didn't spot anything else so I'll go ahead and merge - thanks!

**Pull request merged**